cmd/psa/verify_as_attester.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psa
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/apiclient/verification"
+	"github.com/veraison/evcli/common"
+	"github.com/veraison/go-cose"
+	"github.com/veraison/psatoken"
+)
+
+var (
+	attesterClaimsFile *string
+	attesterKeyFile    *string
+	attesterAPIURL     *string
```

Comment:
```suggestion
	verifierAPIURL     *string
```

Reply:
the convention is to prefix the cmd/psa variables with global scope with a string that is supposed to be unique per file.  Here it's `attester`.

Reply:
If we change L22 as suggested we should change also L20 and L21...